extensions/firefox/content/PdfStreamConverter.jsm

```diff
@@ -956,8 +963,14 @@ PdfStreamConverter.prototype = {
 
     // Create a new channel that is viewer loaded as a resource.
     var ioService = Services.io;
-    var channel = ioService.newChannel(
-                    PDF_VIEWER_WEB_PAGE, null, null);
+    var channel = ioService.newChannel2(PDF_VIEWER_WEB_PAGE,
+                                        null,
+                                        null,
+                                        null,      // aLoadingNode
+                                        Services.scriptSecurityManager.getSystemPrincipal(),
```

Comment:
As pointed out by [Travis](https://travis-ci.org/mozilla/pdf.js/builds/46140824), this line is too long.